IntoHistory/NMapViewController.swift

```diff
@@ -11,21 +11,105 @@ import NMapsMap
 	
 class NMapViewController: UIViewController, CLLocationManagerDelegate {
 
+    // MARK: - Property
+    
     private let courseJSONLoader = LoadingCourseJSON().courses
     var locationManager = CLLocationManager()
+    let infoWindow = NMFInfoWindow()
+    let dataSource = NMFInfoWindowDefaultTextSource.data()
+    
+    // MARK: - View
+    
+    let hStackView: UIStackView = {
+        $0.axis = .horizontal
+        $0.backgroundColor = .white
+        return $0
+    }(UIStackView())
+    
+    let vStackView: UIStackView = {
```

Comment:
p1;

접근제어자 추가 부탁드려요! 아래 let들도 전부 필요해보이지만, 리뷰는 여기까지만 달겠습니다. 수정 부탁드려요! 👍
```suggestion
    private let vStackView: UIStackView = {
```

Reply:
> p1;
> 
> 접근제어자 추가 부탁드려요! 아래 let들도 전부 필요해보이지만, 리뷰는 여기까지만 달겠습니다. 수정 부탁드려요! 👍

6c480ff
수정하였습니다.